mdoc/Mono.Documentation/MDocUpdater.cs

```diff
@@ -1689,9 +1689,27 @@ public void DoUpdateType2 (string message, XmlDocument basefile, TypeDefinition
                     DeleteMember ("Member Removed", output, oldmember, todelete, type);
                     statisticsCollector.AddMetric(typeEntry.Framework.Name, StatisticsItem.Members, StatisticsMetrics.Removed);
                     continue;
-                }
+                }
+
+                // Duplicated
+                   //CheckFunction
+                    Func<XmlNode, XmlElement, Boolean> Dochk = (a, b) =>

```

Comment:
Please change the name of this variable to something a bit more descriptive ... something like `docIdCheck` works :)